dcpy/connectors/socrata/publish.py

```diff
@@ -123,20 +127,28 @@ def from_dataset_attributes(cls, attrs: md.DatasetAttributes):
                 return cls(
                     name=attrs.display_name,
                     description=attrs.description,
-                    tags=attrs.tags,
+                    category=attrs.category or "",
```

Comment:
curious if you have any thoughts on this - should this be nullable? Throw an error?

Reply:
yeah, this should throw an exception in the connector if we don't have those fields. Gonna fix that real quick.